### ISSUE TITLE
Cleaning Codebase 

### ISSUE THREAD

**Issue by saurabhbakolia** [osBins/chatApp#84]

**Comment by saurabhbakolia** [comment]
I want to work on cleaning codebase, please assign issue to me

**Comment by osBins** [comment]
Thank you for wanting to take up this issue @saurabhbakolia! Assigning it to you!

**Comment by iamrahulrnair** [comment]
Hi, may I work on this, and could u specify the description of what exactly the goal, the title seems abstract, should I modularize it?

**Comment by osBins** [comment]
Yes we still need work on this @iamrahulrnair! The current code base is pretty inconsistent in terms of syntax, spaces, commas, semi-colons. Do you have anything in mind?

**Comment by KossakowskaKarolina** [comment]
It would also be helpful to get rid of unused old functions that are commented. And add better comments describing what existing functions do. Some of them are missing that.

**Comment by PrimalPimmy** [comment]
@KossakowskaKarolina Would you like to work on this issue?

**Comment by KossakowskaKarolina** [comment]
@PrimalPimmy Yes, I can work on this issue, just need a few details for clarification. Which files should be included in this assignment? All of the html, css and js files? What should I do with files marked as old? Is there any specification in general that I should follow? Any clues would be appreciated :)

**Comment by PrimalPimmy** [comment]
@KossakowskaKarolina I merged a PR that changes a few files and folder structure. you can update your fork accordingly :3 . Also, it may be nice if any spam comments are removed. I would suggest keep the .old files for now, I'll let you know when they are not needed.

**Comment by KossakowskaKarolina** [comment]
@PrimalPimmy Ok, I think I got it :) Would you assign me to this?

**Comment by PrimalPimmy** [comment]
Sure!
